### PR TITLE
Modify the documentation of padding="same" for conv layer in conv1d.py

### PR DESCRIPTION
**Pull request by tilakrayal**
Modify the documentation of **padding="same"** for conv layer in `conv1d.py`

### CODE REVIEW

File: keras/layers/convolutional/conv1d.py [hunk 1]
```diff
@@ -75,8 +75,8 @@ class Conv1D(Conv):
         any `dilation_rate` value != 1.
       padding: One of `"valid"`, `"same"` or `"causal"` (case-insensitive).
         `"valid"` means no padding. `"same"` results in padding with zeros
-        evenly to the left/right or up/down of the input such that output has
-        the same height/width dimension as the input.
+        evenly to the left/right or up/down of the input.When padding=
```
Comment (haifeng-jin) [inline keras/layers/convolutional/conv1d.py:78]: Insert a space between "input.When".

Comment (haifeng-jin) [inline keras/layers/convolutional/conv1d.py:78]: move padding to the next line and make it like `padding="same"`. Format the line lengths for the rest of the lines accordingly.

**Review by haifeng-jin — CHANGES_REQUESTED** [review]
Thanks for the update!

**Review by haifeng-jin — CHANGES_REQUESTED** [review]
We still have the following lint errors. You can also see it yourself in the CI.
```
keras/layers/convolutional/conv1d.py:78:63: W291 trailing whitespace
keras/layers/convolutional/conv1d.py:79:71: W291 trailing whitespace
```

**Comment by gbaned** [comment]
Hi @tilakrayal Can you please check @haifeng-jin's comments and keep us posted ? Thank you!

**Comment by gbaned** [comment]
Hi @tilakrayal Any update on this PR? Please. Thank you!

**Pull request closed without merging**